Formula/haxe.rb

```diff
@@ -57,6 +57,10 @@ def install
       ENV["OPAMYES"] = "1"
       ENV["ADD_REVISION"] = "1" if build.head?
       system "opam", "init", "--no-setup", "--disable-sandboxing"
+
+      # Remove when the camlp5 dependency is updated to 8.0.0 or later
```

Comment:
Brewed `camlp5` is 8.0.0. Can we use that?

Reply:
Missed this, but I got it to work locally by applying the upstream patch and using `--working-dir`, as pointed out by @cho-m.